frontend/src/components/Login.vue

```diff
@@ -54,14 +54,14 @@
 
 <script>
 import { minLength, required } from 'vuelidate/lib/validators';
+import { mapGetters } from 'vuex';
 
 export default {
   data() {
     return {
       username: '',
       password: '',
       showPass: false,
-      error_msg: '',
       successAlert: false,
       showSpinner: false
```

Comment:
Could we rename for "showLoadingSpinner" or sth to make it more transparent?
Also spinner could be centered as right now it's aligned to the left